src/main/features/claim/routes/pay.ts

```diff
@@ -58,35 +58,7 @@ async function successHandler (res, next) {
   const user: User = res.locals.user
   const externalId: string = draft.document.externalId
 
-  let claimIsAlreadyFullyPersisted: boolean
-
   try {
```

Comment:
Could i please ask, what is the reason earlier to check if claim is already fully persisted? and why we think its not necessary now? This is for my understanding...

Reply:
@KiranVarma02 , we felt that making call is unnecessary as we are having transaction bound call.

Reply:
ok, thanks for explaining